OpenRA.Mods.Common/Traits/Buildings/Reservable.cs

```diff
@@ -65,6 +65,12 @@ public static bool IsReserved(Actor a)
 			return res != null && res.reservedForAircraft != null && !res.reservedForAircraft.MayYieldReservation;
 		}
 
+		public static bool IsAvailableFor(Actor reservable, Actor forUser)
```

Comment:
Can you please change this to `IsAvailableFor(Actor self, Actor forAircraft)` to match the the methods above?

Reply:
I renamed it to forActor, since it's the Actor not the Aircraft (still consistent with the Reserve method). I also duplicated the ReturnToBase change to  HeliReturnToBase.